contracts/StablecoinConverter.sol

```diff
@@ -175,18 +186,35 @@ contract StablecoinConverter is EpochTokenLocker {
                 volumes[i]
             );
         }
+        int fee = tokenConservation[0];
+        require(fee < auctionData[batchIndex].feeCollected, "Fee is not higher than before");
```

Comment:
The way we are calculating the tokenConservation, it will always be smaller than 0. That means the fee is a negative number. We are selling more than we are buying. Is this confusing? Probably yes, let me think of an solution.

Yes, it is a strict inequality